src/task/FinishGatheringICECandidatesTask.ts

```diff
@@ -86,6 +86,14 @@ export default class FinishGatheringICECandidatesTask extends BaseTask {
         this.removeEventListener();
         reject(error);
       };
+      if (!this.context.turnCredentials) {
+        this.context.peer.addEventListener('icegatheringstatechange', () => {
```

Comment:
- How can `this.context.turnCredentials` be `null` or `undefined`? `ReceiveTURNCredentialsTask` in the start operation is always setting this to an `MeetingSessionTURNCredentials` instance.
- How about removing this event listener in `this.removeEventListener` too?

Reply:
* This is only for a debug mode testing with local server so there won't be any turn relay. 
* "How about removing this event listener in this.removeEventListener too", Sure I could do that.